### PR TITLE
Admin cluster tab refresh

### PR DESCRIPTION
**Pull request by ghsolomon**
Hoist P/R Checklist
-------------------

**Pull request authors:** Review and check off the below. Items that do not apply can also be
checked off to indicate they have been considered. If unclear if a step is relevant, please leave
unchecked and note in comments.

- [x] Caught up with `develop` branch as of last change.
- [x] Added CHANGELOG entry, or determined not required.
- [x] Reviewed for breaking changes, added `breaking-change` label + CHANGELOG if so.
- [x] Updated doc comments / prop-types, or determined not required.
- [x] Reviewed and tested on Mobile, or determined not required.
- [x] Created Toolbox branch / PR, or determined not required.

**If your change is still a WIP**, please use the "Create draft pull request" option in the split
button below to indicate it is not ready yet for a final review.

> **Pull request reviewers:** when merging this P/R, please consider using a *squash commit* to
> collapse multiple intermediate commits into a single commit representing the overall feature
> change. This helps keep the commit log clean and easy to scan across releases. PRs containing a
> single commit should be *rebased* when possible.

### CODE REVIEW

**Comment by ghsolomon** [comment]
New `errorMessageBanner` (local to the `clusterTab` component for now):

Screenshot 2024-09-11 at 2 14 18 PM

**Comment by ghsolomon** [comment]
@lbwexler switched to using `errorMessage` cmp

File: admin/tabs/cluster/ClusterTabModel.ts [hunk 1]
```diff
@@ -64,20 +69,22 @@ export class ClusterTabModel extends HoistModel {
...
     }
 
     constructor() {
         super();
+        makeObservable(this);
 
         this.timer = Timer.create({
-            runFn: this.autoRefreshAsync,
+            runFn: () => {
+                if (this.tabModel?.isActive) this.autoRefreshAsync();
```
Comment (amcclain) [inline admin/tabs/cluster/ClusterTabModel.ts:86]: Ah, that's smart, vs using visibility

**Comment by amcclain** [comment]
(imagine it's the load balancer that's having the issue - every other request goes to a blackhole)

**Comment by lbwexler** [comment]
I think my proposal splits the difference -- it preserves maximal state,
and just masks things temporarily.

We could eventually replace the mask with the banner, but I think my
solution is less jarring

**Pull request closed without merging**